robottelo/config/validators.py

```diff
@@ -6,7 +6,15 @@
 
 validators = dict(
     server=[
-        Validator("server.hostname", must_exist=True),
+        Validator("server.hostname", must_exist=False),
+        Validator("server.hostnames", must_exist=True, is_type_of=list),
+        Validator("server.version.release", must_exist=True),
+        Validator("server.version.source", must_exist=True),
+        Validator(
+            "server.xdist_behavior", must_exist=True, is_in=['run-on-one', 'balance', 'on-demand']
+        ),
+        Validator("server.on_demand", must_exist=True, is_type_of=bool),
```

Comment:
I think this field is out of date with the design now?

Reply:
ahhhh you're right!

Reply:
fixed